ignition-rendering/PKGBUILD

```
@@ -8,8 +8,8 @@ engines. It offers unified APIs for creating 3D graphics applications."
 arch=('x86_64')
 url="https://ignitionrobotics.org/libs/rendering"
 license=('Apache')
-depends=('ignition-plugin')
-makedepends=('cmake')
+depends=('ignition-common' 'ignition-math' 'ignition-plugin' 'ogre-1.9')
```

```suggestion
depends=('ignition-common' 'ignition-math' 'ignition-plugin' 'ogre=1.9')
```

Just a little bit cleaner.

---

I went with `=` as you suggested as ignition-rendering would not build with Arch’s `ogre` package yet.